<?xml version="1.0"?>
<launch>
    <!-- Simulator Bridge -->
    <include file="$(find styx)/launch/server.launch" />

    <!--DBW Node -->
    <include file="$(find twist_controller)/launch/dbw_sim.launch"/>

    <!--Waypoint Loader -->
    <include file="$(find waypoint_loader)/launch/waypoint_loader.launch"/>

    <!--Waypoint Follower Node -->
    <include file="$(find waypoint_follower)/launch/pure_pursuit.launch"/>

    <!--Waypoint Updater Node -->
    <include file="$(find waypoint_updater)/launch/waypoint_updater.launch"/>

    <!--Traffic Light Detector Node -->
    <include file="$(find tl_detector)/launch/tl_detector.launch"/>

    <!--Traffic Light Locations and Camera Config -->
    <param name="traffic_light_config" textfile="$(find tl_detector)/sim_traffic_light_config.yaml" />

    <!--Save Camera Images for Training -->
    <param name="save_images" value="false" type="bool" />

    <!--Use Ground Truth -->
    <param name="use_ground_truth" value="true" type="bool" />

    <!--Distance ahead of vehicle to check for lights -->
    <param name="check_light_distance" value="100.0" type="double" />

    <!--Skip Images Count -->
    <param name="skip_images" value="0" type="int" />
</launch>
